layer-publisher/tests/e2e/happy-case.test.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * Test layer
+ *
+ * @group e2e
+ * 
+ */
+
+import * as cdk from 'aws-cdk-lib';
+import { Stack } from 'aws-cdk-lib';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import * as LayerPublisher from '../../src/layer-publisher-stack';
+import { deployStack, destroyStack } from './utils/cdk-cli';
+import { generateUniqueName, invokeFunction } from './utils/e2eUtils';
+import { LEVEL } from './utils/InvocationLogs';
+
+const runtime = lambda.Runtime.ALL.find((r) => r.name === process.env.RUNTIME) ?? lambda.Runtime.NODEJS_14_X;
+
+const powerToolsPackageVersion = '1.0.1';
```

Comment:
Is this synced to the actual version published on NPM? If yes, how do we increment it when we release?

Reply:
no it's only to test and therefore on purpose set as a lower version than latest

Reply:
in release workflow it takes latest from latest tag https://github.com/awslabs/aws-lambda-powertools-typescript/pull/884/files#diff-39e8ac1723228023ce8c44a0a27a9ce7db36f9ddaf8404cbf2e16c648ad42705R42

Reply:
Perfect, thanks for the explanation.

Leaving the thread open so others can see the answer in case they have the same question